python/lsst/ts/ataos/ataos_csc.py

```diff
@@ -643,6 +692,13 @@ async def do_applyCorrection(self, data: type_hints.BaseMsgType) -> None:
 
         azimuth = data.azimuth % 360.0
         elevation = data.elevation
+        # Set temperature to the fallback temperature if
+        # there is no data.temperature
+        temperature = (
+            data.temperature
```

Comment:
You need to make this change backwards compatible. The way it is we won't be able to update the CSC at the summit until the next xml
release. 

To do this, check that data has the temperature attribute.